rrdp_iostat.ml

```diff
@@ -422,6 +450,10 @@ module Stats_value = struct
 		let (++) = Int64.add in
 		List.fold_left (fun acc v ->
 			{
+				rd_bytes = acc.rd_bytes ++ v.rd_bytes;
+				rd_avg_usecs = acc.rd_avg_usecs ++ v.rd_avg_usecs;
```

Comment:
Should these values be added, or do we just want the maximum?

[Here](https://github.com/xapi-project/xcp-rrdd/commit/f10769a46acd4b9ee17b16bc47461cbee184215b#diff-aae74f219b1c9bcd5b3de6bc44497519L472) it looks like we used to use the maximum.